bundles/org.openhab.binding.vesync/src/main/java/org/openhab/binding/vesync/internal/handlers/VeSyncDeviceAirHumidifierHandler.java

```diff
@@ -171,47 +224,44 @@ public void handleCommand(final ChannelUID channelUID, final Command command) {
                                 new VeSyncRequestManagedDeviceBypassV2.EnabledPayload(command.equals(OnOffType.ON)));
                         break;
                     case DEVICE_CHANNEL_WARM_ENABLED:
-                        logger.warn("Warm mode API is unknown in order to send the command");
+                        logger.warn("{}", getLocalizedText("warning.device.warm-mode-unsupported"));
                         break;
                 }
             } else if (command instanceof QuantityType quantityCommand) {
                 switch (channelUID.getId()) {
                     case DEVICE_CHANNEL_CONFIG_TARGET_HUMIDITY:
                         int targetHumidity = quantityCommand.intValue();
-                        if (targetHumidity < 30) {
-                            logger.warn("Target Humidity less than 30 - adjusting to 30 as the valid API value");
-                            targetHumidity = 30;
-                        } else if (targetHumidity > 80) {
-                            logger.warn("Target Humidity greater than 80 - adjusting to 80 as the valid API value");
-                            targetHumidity = 80;
+                        if (targetHumidity < MIN_TARGET_HUMIDITY) {
+                            logger.warn("{}", getLocalizedText("warning.device.humidity-under", MIN_TARGET_HUMIDITY));
```

Comment:
Why use the formatter (see others too)
```suggestion
                            logger.warn(getLocalizedText("warning.device.humidity-under", MIN_TARGET_HUMIDITY));
```

Reply:
The reason it has this rather weird way of doing it is becausee if I use this:
logger.warn(getLocalizedText("warning.device.unexpected-resp-for-air-purifier"));
the build will output:
[ERROR] org.openhab.binding.vesync.internal.handlers.VeSyncDeviceAirPurifierHandler.java:[432]
Format should be constant. Use placeholder to reduce the needless cost of parameter construction. see http://www.slf4j.org/faq.html#logging_performance

Reply:
@lsiepel regarding this is there some other trick I'm not aware of, to clear that error without having to add the formatter?